### ISSUE TITLE
iframe elements integration

### ISSUE THREAD

**Issue by jt151077** [enableffs/enableffs.github.io#202]
consider whether the iframe interactive objects should be replaced / integrated to allow better accessibility. If not possible, provide an alternative descriptive widget for screen-reader support
